cypress/support/step_definitions/event.js

```diff
@@ -5,3 +5,24 @@ When('I click on {string}', (templateSelector) => {
   const selector = nunjucks.renderString(templateSelector, cy.context);
   cy.get(selector).click();
 });
+
+When('I double click on last element of {string}', (templateSelector) => {
+  const selector = nunjucks.renderString(templateSelector, cy.context);
+  cy.get(selector).last().dblclick();
+});
+
+When('I double click on elements of {string}', (templateSelector) => {
+  const selector = nunjucks.renderString(templateSelector, cy.context);
+  cy.get(selector).click({ multiple: true }).dblclick({ multiple: true });
```

Comment:
why click before double click ? 

Reply:
I had trouble targetting a specific element.
Done - remove this rule